eclair-core/src/main/scala/fr/acinq/eclair/Eclair.scala

```diff
@@ -67,19 +67,22 @@ trait Eclair {
 class EclairImpl(appKit: Kit) extends Eclair {
 
   implicit val ec = appKit.system.dispatcher
-  implicit val timeout = Timeout(60 seconds) // used by akka ask
+  implicit val timeout = Timeout(appKit.nodeParams.apiTimeout) // used by akka ask
```

Comment:
It feels wrong here (see comment below). Adding an implicit timeout parameter to each method call in the `Eclair` interface would be cleaner

Reply:
This is what it would be in Service.scala:

```scala
          formFields(timeout.?) { tm_opt =>
            implicit val timeout = tm_opt.getOrElse(Timeout(30 seconds))
            withRequestTimeout(timeout.duration + 2.seconds) {
              withRequestTimeoutResponse(timeoutResponse) {
```